app/test/frontend/handlers/_utils.dart

```diff
@@ -34,12 +34,22 @@ Future<shelf.Response> issueGetUri(Uri uri) async {
 }
 
 Future<String> expectHtmlResponse(shelf.Response response,
```

Comment:
Could we define:
```dart
Future<String> expectHtmlResponse(String path, {
  int status,
  List<Pattern> present = [],
  List<Pattern> absent = [],
}) {
  ...
}
```

Or maybe that's something we do later..

Reply:
Do you want to also invoke the handler via path? Then it would be a different for the different services.

I'm also not sure what status would be outside of 200?